ansible_collections/arista/avd/roles/eos_designs/templates/overlay/logic-evpn-route-servers.j2

```diff
@@ -13,3 +14,19 @@
 {%         endif %}
 {%     endif %}
 {% endfor %}
+{% for evpn_gateway_vxlan_route_server in switch.evpn_gateway_remote_peers | arista.avd.natural_sort %}
```

Comment:
Same here as above, you still use ...route_server. Suggest changing to something more descriptive to make it less confusing.